pkg/runtime/conversion_generator.go

```diff
@@ -302,11 +302,15 @@ func (g *conversionGenerator) generateConversionsForSlice(inType, outType reflec
 }
 
 func (g *conversionGenerator) generateConversionsForStruct(inType, outType reflect.Type) error {
+	errs := []string{}
```

Comment:
@pweil - this method is only used to decide whether we want to generate conversion or not - if you return an error from it, conversion will not be generated.
So IIUC, this is not exactly what you expect.